cleanlab/cleanlab-studio
# Feature: add support for project deletion from python api

This PR adds a method to the Python API for deleting projects

### How to test
Test w/ backend branch `feat/python-api-project-deletion`
- Check that deleting one of your projects succeeds
- Check that attempting to delete project w invalid ID/project you don't own fails
